core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1209,6 +1214,24 @@ public String toString() {
           .build();
 
   //
+  // Log server related properties
+  //
+  public static final PropertyKey LOG_SERVER_BASE_LOGS_DIR =
+      new Builder(Name.LOGSERVER_BASE_LOGS_DIR)
+          .setDefaultValue(String.format("${%s}/logs", Name.WORK_DIR))
```

Comment:
Will this default conflict with `alluxio.logs.dir` value? Is that intentional?

Reply:
The two properties can be the same. Since log server will create subdirectories, e.g. master_logs, worker_logs, proxy_logs, secondary_master_logs in LOB_SERVER_BASE_LOGS_DIR, and it will include the hostname in the log file name, therefore it will not conflict with alluxio.logs.dir.